app/models/member_permission.rb

```diff
@@ -1,5 +1,5 @@
 class MemberPermission < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, touch: true
```

Comment:
Why? 🤔

Reply:
This originally was used for `updated_at`, which I later removed: https://github.com/3scale/porta/pull/121/commits/8bc646f79aaa5d2a2a24b5cd406313203ec61e47

So, member permissions representer delegated `updated_at` to its user. But still, this method is not very precise. As, for example, if some user properties change (name, email etc.), permissions' `updated_at` would change also, and it's not true. So, I thought that it's better remove it to avoid confusion.

I think it's still useful, as when you change user permissions, the `updated_at` of the users change, which seems logical to me - as it's a user's property.